.github/workflows/sync_release_pr.yml

```diff
@@ -0,0 +1,26 @@
+name: "Sync Release to Main"
+
+on:
+  push:
+    branches:
+      - 'release'
```

Comment:
Wait, I'm confused about the flow again. How can there be a single release branch that multiple packages are doing releases from while still supporting the async PR approval model? The whole point of using a branch for a release is that the branch can be a snapshot of the state that we want to release for a given package version.

@chunhtai is there a diagram or flow chart in a doc somewhere that shows the final plan for how the branches will work?

Reply:
https://docs.google.com/document/d/1jUoFaawutbYsCI5oY3pDP_l-xpv6FhDKlcI1-EoT02s/edit?tab=t.0